subsys/bluetooth/controller/hci/hci_driver.c

```diff
@@ -105,15 +131,23 @@ static void prio_recv_thread(void *p1, void *p2, void *p3)
 			 */
 			ll_rx_dequeue();
 
-			/* Send the rx node up to Host thread, recv_thread() */
-			BT_DBG("RX node enqueue");
-			k_fifo_put(&recv_fifo, node_rx);
+			buf = process_prio_evt(node_rx);
+			if (buf) {
+				bt_recv_prio(buf);
+			} else {
```

Comment:
Do we need a k_yield in the if-clause (inline with after num complete was sent to host?

Reply:
I don't think so. This is not in a loop unlike the num complete, and it's unlikely that we will starve any other threads here given that we only process a very small subset of events